wiki/Contests/Mapping_Olympiad/en.md

```diff
@@ -0,0 +1,83 @@
+# osu! Mapping Olympiad
+
+![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)
+
+**osu! Mapping Olympiad** is an all-gamemode official contest line-up that was originally from a project of official beatmapping contests revival in 2018. Its particularity is to explore different contest concepts, going from traditional mapping contests to speedmapping or aspire, to mention just a few. 
```

Comment:
both sentences read unnaturally, not to mention [excessive usage of "contest"](https://tc.s-ul.eu/QMfyL3AX.png)